transaction/models.py

```diff
@@ -16,7 +16,7 @@ class Transaction(models.Model):
         related_name="receiver",
         to_field="name",  # noqa E501
     )
-    transfer_amount = models.FloatField()
-    commission = models.FloatField()
-    status = models.CharField(choices=STATUS, max_length=100)
+    transfer_amount = models.DecimalField(max_digits=12, decimal_places=2)
+    commission = models.DecimalField(max_digits=12, decimal_places=2, default=0)
+    status = models.CharField(choices=STATUS, max_length=100, default='PAID')
     timestamp = models.DateTimeField(auto_now_add=True)
```

Comment:
please add `editable` attr to the `commission`, `status` and `timestamp` fields and set it to `False`
also add default value for the `transfer_amount` equals to 0

Reply:
Done